django/http/request.py

```diff
@@ -61,6 +61,7 @@ def __init__(self):
         self.method = None
         self.resolver_match = None
         self._post_parse_error = False
+        self.content_type = self.content_params = None
```

Comment:
I see the parallel to how they are assigned in `WSGIRequest`, but still I would put them on separate lines here.